src/ghdcbot/adapters/github/rest.py

```
@@ -0,0 +1,979 @@
+                issue_numbers.append(issue["number"])
+                issue_events.extend(self._issue_events(owner, repo, issue, since))
+        return issue_events, issue_numbers
+
+    def _collect_pull_request_events(
+        self, owner: str, repo: str, since: datetime
+    ) -> tuple[list[ContributionEvent], list[int], int]:
+        pr_events: list[ContributionEvent] = []
+        pr_numbers: list[int] = []
+        pr_opened_count = 0
+        # Track merged PRs to detect reverts
+        merged_prs: dict[int, dict] = {}  # pr_number -> pr data
```

Ruff (0.14.14)

[error] Local variable `merged_prs` is assigned to but never used

Remove assignment to unused variable `merged_prs`

(F841)